elytron/src/main/java/org/wildfly/extension/elytron/KeyStoreService.java

```diff
@@ -176,6 +176,11 @@ public void start(StartContext startContext) throws StartException {
                         keyStore = AtomicLoadKeyStore.atomize(detected);
                     }
                 } else {
+                    if (keyStore == null) {
+                        ROOT_LOGGER.keyStoreNotExistsDefaultWillBeUsed();
```

Comment:
See what the opinion of others is but I think we need to start asking if we really do need one more log message which will be on the console.  We are already receiving bug reports for the verbosity of our current messages so worth checking if any new ones are really justified.

As this is a one time event I think this could safely be logged at DEBUG level, this would mean we don't need the error to be defined in ElytronSubsystemMessages - if would be good for the message to also contain the type that was used.

Reply:
I agree with Darran.
WARN is too high as this behaviour will not prevent the server to work. 
Besides, the log in itself does not contain enough contextual information to let the user "fix" the problem. Which keystore is impacted? What's "Used default" means: to us a "default keystore" (if such thing exists) or that a new keystore with "default type" will be created? What's the default type in that case?

Reply:
Hi @darranl @jmesnil, thank you for your suggestions.  I have added the requested changes.